From d533dd3e954400d376ea33c2104e077fb108f64b Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 3 May 2023 14:24:17 +0200
Subject: [PATCH] arm64: defconfig: Enable Moortec DTS sensor support

This patch adds in built-in the Moortec DTS sensor driver
used by STM32MP SOCs.

Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/305107
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Change-Id: Ic77b2b635953cb45cd7ecb0ee0dfb95945de3744
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/390200
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -67,6 +67,7 @@ CONFIG_RPMSG_IRQ=m
 CONFIG_RPMSG_TTY=m
 CONFIG_RPMSG_VIRTIO=m
 CONFIG_RTC_DRV_STM32=y
+CONFIG_SENSORS_MR75203=y
 CONFIG_SAMPLES=y
 CONFIG_SAMPLE_RPMSG_CLIENT=m
 CONFIG_SND_SOC_STM32_I2S=m
